src/Helper/Service/AssetFactory.php

```diff
@@ -12,6 +12,10 @@
 
 use function is_array;
 
+/**
+ * @final
+ * @psalm-suppress DeprecatedInterface Compatibility with Service Manager 2 should be removed in version 3.0.
```

Comment:
I think the dependency to laminas-servicemanager can also be removed in version 3.

The same applies to all other factories:

https://github.com/laminas/laminas-view/blob/5e8d8002811234e99dbaf021ab62007b5cfc22b3/src/Helper/Service/FlashMessengerFactory.php#L12
https://github.com/laminas/laminas-view/blob/5e8d8002811234e99dbaf021ab62007b5cfc22b3/src/Helper/Service/IdentityFactory.php#L14

See the factory for the Doctype helper:

https://github.com/laminas/laminas-view/blob/3855f6ef2c7f5c514ba0c0e9a1876e4f42ec8384/src/Helper/Service/DoctypeFactory.php#L7-L12

Reply:
Absolutely - and we can also say bye-bye to Interop too.